Gordon360/Models/ViewModels/HousingPreferenceViewModel.cs

```diff
@@ -0,0 +1,19 @@
+using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels;
+
+namespace Gordon360.Models.ViewModels
+{
+    public class HousingPreferenceViewModel
+    {
+        public string ApplicationID { get; set; }
+        public string Preference1 { get; set; }
+        public static implicit operator HousingPreferenceViewModel(Preference p)
```

Comment:
Same here, what is the benefit of duplicating this database model as a ViewModel?